model_api/python/model_api/models/classification.py

```diff
@@ -371,15 +383,15 @@ def resolve_labels(self, predictions):
         def get_predecessors(lbl, candidates):
             """Returns all the predecessors of the input label or an empty list if one of the predecessors is not a candidate."""
             predecessors = []
-            last_parent = self._get_parent(lbl)
+            last_parent = self.label_tree.get_parent(lbl)
             if last_parent is None:
                 return [lbl]
 
             while last_parent is not None:
                 if last_parent not in candidates:
                     return []
                 predecessors.append(last_parent)
-                last_parent = self._get_parent(last_parent)
+                last_parent = self.label_tree.get_parent(last_parent)
 
             if predecessors:
                 predecessors.append(lbl)
```

Comment:
This diff looks weird, since the line `last_parent = self._get_parent(last_parent)` doesn't actually exists in the PR. I'd avoid applying this to prevent merge mistakes.